packages/fxa-payments-server/src/store/reducers.ts

```diff
@@ -21,18 +32,24 @@ import {
   resetUpdatePayment,
 } from './actions';
 
-export const defaultState: State = {
+export const defaultState = {
```

Comment:
And here where `defaultState` is constructed, we can fully specify the shape and initial value for the Redux state in one spot.